__test__/db.mocks.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const syllabusesSchema = require('../db/data/syllabusesSchema');
+
+const connect = () => new Promise((resolve, reject) => {
+  mongoose.connect('mongodb://localhost:27017/test', { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
I can see that this is the mocked db used for testing right?

Reply:
@phucciTr So this is actually using the same syllabus schema as `prod`, but yes importing it here allows for easier testing - I'm using the same schema, just a mocked/test version of the `connect()` function found in `db/index.js`.